System/Clock.hsc

```diff
@@ -141,7 +141,11 @@ clockToConst ProcessCPUTime = #const CLOCK_PROCESS_CPUTIME_ID
 clockToConst  ThreadCPUTime = #const CLOCK_THREAD_CPUTIME_ID
 #if defined (linux_HOST_OS)
 clockToConst    MonotonicRaw = #const CLOCK_MONOTONIC_RAW
+#if defined(CLOCK_BOOTTIME)
 clockToConst        Boottime = #const CLOCK_BOOTTIME
+#else
+clockToConst        Boottime = 7
```

Comment:
It's defined on Linux in: _/usr/include/bits/time.h_ and _/usr/include/linux/time.h_
`/usr/include/linux/time.h:#define CLOCK_BOOTTIME			7`

I checked it's history and meanining from different sources and I'm sure it has
such value for all Linux kernels from version 2.6.39 where it was introduced.